recipes/webdataset/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "webdataset" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/webdataset-{{ version }}.tar.gz
+  sha256: d7204f3615b14c9b6ad44570858dcf32ea0c4e07bd32ec2ac346e95528ac087b
+
+build:
+  skip: true  # [py2k or win]  # can't include win because of pytorch
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - braceexpand
+    - numpy
+    - python
```

For noarch need to specify Python version range:
```suggestion
    - python >=3.6
  run:
    - braceexpand
    - numpy
    - python >=3.6
```